handlers/firebase/handler.go

```diff
@@ -286,6 +287,10 @@ func (h *handler) sendWithCredsJSON(msg courier.MsgOut, res *courier.SendResult,
 
 		payload.Message.Token = msg.URNAuth()
 		payload.Message.Android.Priority = "high"
+		
+		if quickReplies := msg.QuickReplies(); quickReplies != nil {
+			payload.Message.Data.QuickReplies = string(jsonx.MustMarshal(quickReplies))
```

Comment:
I understand you chose to expect a string with the quick replies joined, however this PR is not quite right as the quick replies here would be sent for each part of the message.
We want that on the last part only